### DISCUSSION TITLE
Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process.

### DISCUSSION THREAD

**Discussion by franciscoguerra** [balta-io/forum#129] [Tira Dúvidas] [answered]
Curso -> Fundamentos do ASP.NET 6
Aula-> Testando a API

Seguindo os passos da aula quando, os primeiros passo dar tudo ok, status 200 no Postman, quando se pede para usar o CONTINUAR o Postman aparece com esse erro.  Status 500.

nas pesquisa que fiz sobre o erro a maioria está relacionada:
             TLS 1.2
             Microsoft.Data.SqlClient ( no caso esse é o novo pacote)

![erro3](https://user-images.githubusercontent.com/12381369/204138519-e00f412c-895a-4d10-a396-c1e02de38698.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @franciscoguerra, como vai?

Aqui está o link com a solução do problema:
https://balta.io/blog/sql-server-docker

No fim do artigo tem uma sessão chamada **ERROS COMUNS**
